repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -163,6 +163,38 @@ export const getServicesByNocCodeAndDataSource = async (nocCode: string, source:
     }
 };
 
+export const getFaresServicesByNocCodeAndDataSource = async (
```

Comment:
Instead of copy and pasting an existing DB query and adding 1 additional piece of information to select, cant we just edit the current DB query (getServicesByNocCodeAndDataSource) and edit the interface ServiceType to include endDate?

Reply:
Also we presumably need to use the 'convertDateFormat' function on the end date, if its there, like we do with the start date in the query results.

Reply:
getServicesByNocCodeAndDataSource -> gives the list of active services (whose endDate is either not present and if present should be greater than current date)

getBodsOrTndsServicesByNoc -> gets all the services.

Reply:
I understand what the DB queries do, but I dont understand why that means why we cant do what i said above?

Reply:
@NitishAroraKPMG question remains - why do we need to add a new DB query? Why can we not just edit the ones used before?

Reply:
will be deleting the function in next push as it is not referenced anywhere